python/lsst/sconsUtils/utils.py

```diff
@@ -145,15 +147,22 @@ def runExternal(cmd, fatal=False, msg=None):
             msg = "Error running %s" % cmd.split()[0]
         except Exception:
             msg = "Error running external command"
-    process = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    stdout, stderr = process.communicate()
-    if process.returncode != 0:
+
+    # Run with shell unless given a list of options
+    shell = True
+    if isinstance(cmd, (list, tuple)):
+        shell = False
+
+    try:
+        retval = subprocess.run(cmd, shell=shell, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                check=True, text=True)
```

Comment:
Wait, I thought 3.6 doesn't support `text`.  You need `universal_newlines`, I think.